neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/remote/bolt/impl/BoltNeo4jClient.java

```diff
@@ -35,13 +38,15 @@ public BoltNeo4jClient(RemoteNeo4jDatabaseIdentifier identifier, RemoteNeo4jConf
 
 	private Driver createNeo4jDriver(RemoteNeo4jDatabaseIdentifier identifier, RemoteNeo4jConfiguration configuration) {
 		String uri = identifier.getDatabaseUri();
+		Config.ConfigBuilder configBuilder = Config.build();
+		configBuilder.withMaxIdleSessions( configuration.getClientRestPoolSize()  );
```

Comment:
Do these two attibutes have the same meaning?

Reply:
Hm ... What do you mean ? Code in row 42 is correct default configuration of Bolt driver.

Reply:
This line is setting the MaxIdleSessions (a time) using the value in ClientRestPoolSize ( a number of connections). I don't understand how the two values are connected.

Reply:
Hm ... I am look to Javadoc
/**
         * The max number of idle sessions to keep open at once. Configure this
         * higher if you want more concurrent sessions, or lower if you want
         * to lower the pressure on the database instance.
         *
         * @param size the max number of idle sessions to keep open
         * @return this builder
         */
        public ConfigBuilder withMaxIdleSessions( int size )

It is not a time ... it is count of connections.

Reply:
Ah, OK. I probably checked the wrong property. But they still seems quite different to me. One is a pool size and the other one is a limit. I think it's better to keep two separates properties for this. Let's ignore bolt for this PR.

Reply:
This method is still called `getClientRestPoolSize()`, could you rename it to `getClientPoolSize()`